proto/api.proto

```diff
@@ -241,6 +241,25 @@ message ScanResponse {
   repeated KeyValue rows = 2 [(gogoproto.nullable) = false];
 }
 
+// A ReverseScanRequest is the argument to the ReverseScan() method. It specifies the
+// start and end keys for the scan and the maximum number of results.The InclusiveKey
+// and InclusiveEndKey flags are used to indicate whether the results need to include
+// the start key and end key.
+message ReverseScanRequest {
+  optional RequestHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
+  // Must be > 0.
+  optional int64 max_results = 2 [(gogoproto.nullable) = false];
+  optional bool inclusive_key = 3 [(gogoproto.nullable) = false];
+  optional bool inclusive_end_key = 4 [(gogoproto.nullable) = false];
```

Comment:
These two flags are only used internal for cross multiple ranges scan.To user the start key is always exclusive and end key is inclusive.Maybe I need add a `InternalReverseScanRequest` to include the flags and be not exposured to user.